build.sbt

```diff
@@ -446,7 +476,7 @@ lazy val storage = project
 
 lazy val delta = project
```

Comment:
project should aggregate on all submodules, like rdf for example?

Reply:
you mean the `lazy val root` ?

I don't understand

Reply:
shouldn't the delta project have an aggregate statement (`.aggregate(rdf, sourcing)`)?